tests/core/_utils/test_filesystem.py

```diff
@@ -0,0 +1,28 @@
+import json
+
+import pytest
+
+from ethpm_cli._utils.filesystem import atomic_replace
+
+
+@pytest.fixture
+def original(tmp_path):
+    original = tmp_path / "original.json"
+    original.write_text(json.dumps({"one": "two"}))
```

Comment:
you could simplify these tests by dropping the json loading/dumping and just writing/reading basic strings to/from the files and asserting they are present/missing